File: 2/1_Cuatrimestre/Programacion_y_Diseño_Orientado_a_Objetos/JavaFinal/NapakalakiJavaSinInterfaz/src/NapakalakiGame/DeathBadConsequence.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author Javi
 */
public class DeathBadConsequence extends NumericBadConsequence{
    DeathBadConsequence(String text){
        super(text,Player.MAXLEVEL,BadConsequence.MAXTREASURES,BadConsequence.MAXTREASURES);
    } 
}

File: 2/1_Cuatrimestre/PDOO/RubyFinal/DeathBadConsequence.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module NapakalakiGame
  class DeathBadConsequence < NumericBadConsequence
    def initialize(text)
      super(text,Player.MAXLEVEL,BadConsequence.MAXTREASURES,BadConsequence.MAXTREASURES)
    end
    def isEmpty()
      isEmpty
    end
  end
end
